packages/graphql/src/schemaTypes/objectTypes/gql-Mutation.ts

```diff
@@ -33,7 +33,7 @@ export const mutation = mutationType({
       },
       resolve: async (_, args, ctx) => {
         ctx.actions.error.clearError(args.id)
-        await ctx.lifecycleManager.refreshLifecycle()
+        await ctx.lifecycleManager.refreshLifecycle().catch(ctx.lifecycleManager.onLoadError)
```

Comment:
This is why the config error was getting set to base error rather than project error (and why the title was disappearing). The first time around, `lifecycleManager.onLoadError` was called when the configuration error was thrown.

However, the second time around (after clicking "try again"), it was called here, but wasn't caught so it went up to base error with no title. Now that we're catching it here, it stays at the current project level and has the correct title.

Reply:
Nice job tracking this down.

Reply:
I have @ZachJW34 to thank for helping me track it